Do not overwrite *FLAGS
Use CC and PREFIX from the environment

--- a/config.mk
+++ b/config.mk
@@ -4,13 +4,8 @@
 # Customize below to fit your system
 
 # paths
-PREFIX = /usr/local
 MANPREFIX = $(PREFIX)/share/man
 
 # flags
-CPPFLAGS = -DVERSION=\"$(VERSION)\" -D_DEFAULT_SOURCE -D_XOPEN_SOURCE=700 -D_BSD_SOURCE
-CFLAGS   = -std=c99 -pedantic -Wall -Wextra -Os
-LDFLAGS  = -lpthread -s
-
-# compiler and linker
-CC = cc
+CPPFLAGS += -DVERSION=\"$(VERSION)\" -D_DEFAULT_SOURCE -D_XOPEN_SOURCE=700 -D_BSD_SOURCE
+LDFLAGS  += -lpthread
